grtc/app.js

```diff
@@ -0,0 +1,471 @@
+'use strict';
```

Comment:
We shouldn't need `'use strict';` with Babel (it is added automatically)

Reply:
Would it be possible to split this file up, perhaps into one-file-per-class?

Reply:
`use strict` removed, I will split into files next.